TravelAgency-spring-mvc/src/main/webapp/WEB-INF/tags/pageTemplate.tag

```diff
@@ -60,10 +60,13 @@
                         <c:if test="${empty authenticatedUser}">
                             <li><my:a href="/customer/registration"><f:message key="navigation.customer.registration"/></my:a></li>
                         </c:if>
-                        <!--<li><my:a href="/customer/edit"><f:message key="navigation.customer.edit"/></my:a></li>-->
+
                         <c:if test="${not empty authenticatedUser}">
                             <li><my:a href="/auth/logout"><f:message key="navigation.customer.logout"/></my:a></li>
                         </c:if>
+                        <c:if test="${not empty authenticatedUser}">
+                            <li><my:a href="/customer/edit/${authenticatedUser.getId()}"><f:message key="customer.myAccount"/></my:a></li>
```

Comment:
This is not really a safe approach. Now anyone could edit any customer by putting id to the address. I suggest obtaining the ID from the authenticated user by the controller. (the c:if can stay)